dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/publish/servers/TxFaultToleranceReordered/jvm.options

```diff
@@ -0,0 +1,2 @@
+# Run the Fault Tolerance interceptors before the Transactional ones
+-Dcom.ibm.ws.faulttolerance.run.before.transactional=true
```

Comment:
Putting `com.ibm.ws.faulttolerance.run.before.transactional=true` in the bootstrap.properties file is the preferred method i believe

Reply:
Test configuration changed to use bootstrap.properties as suggested.